pysat/instruments/cnofs_vefi.py

```diff
@@ -68,7 +94,7 @@ def clean(inst):
 
     Parameters
     -----------
-    inst : (pysat.Instrument)
+    self : (pysat.Instrument)
```

Comment:
Why did this change to self?

Reply:
Good catch.  Probably a copy and paste issue from somewhere.  Fixed.